### ISSUE TITLE
Add an option to hide localities

### ISSUE THREAD

**Issue by ldsvrn** [organicmaps/organicmaps#6960]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.

In some areas with a lot of localities defined, the map can get quickly filled with useless information, such as names of farming fields.

![Screenshot_20231224-175847.png](https://github.com/organicmaps/organicmaps/assets/70917976/f79f6c44-c501-4844-8e23-e62bc392d653)

**Describe the ideal solution**
A clear and concise description of what you want to see in Organic Maps.

I could be great of we could hid theses "locality"  from the map.

**Describe alternatives you have considered**
- How do you solve this issue now with Organic Maps or other apps?
- Attach any examples, screenshots, or screen recordings from other apps that help us to better understand the idea.

I usually zoom out until they disappear.

**Additional context**

**Comment by fizwidget** [comment]
I’ve also noticed cases where localities clutter up the map a bit. E.g. at Cockatoo Island in Sydney, the localities within the island hide the name of the island itself (which is really more important):

![IMG_2765](https://github.com/organicmaps/organicmaps/assets/1172012/54441935-825f-4311-ae28-5af000531d97)

**Comment by pastk** [comment]
- possible solution: https://github.com/organicmaps/organicmaps/issues/6223

**Comment by pastk** [comment]
In the meantime we might reduce visibility of localities by one zoom level.

**Comment by pastk** [comment]
> I’ve also noticed cases where localities clutter up the map a bit. E.g. at Cockatoo Island in Sydney, the localities within the island hide the name of the island itself (which is really more important):
> 

While I agree that `place=islet` should be generally higher than a locality
(but maybe there could be groups of islets tagged together as a locality??),

here could be a case of mistagging also, as [`place=locality`](https://wiki.openstreetmap.org/wiki/Tag:place%3Dlocality) is used for unpopulated places only.

**Comment by fizwidget** [comment]
> here could be a case of mistagging also, as [`place=locality`](https://wiki.openstreetmap.org/wiki/Tag:place%3Dlocality) is used for unpopulated places only.

Yeah that could well be the case here, I’ll have more of a think about it.

It is “unpopulated” in the sense that nobody lives there - it’s essentially an open air museum of unused historic buildings. Maybe there’s a better way of tagging the precincts within it…

**Comment by ldsvrn** [comment]
Is there a way to simply completely hide theses?
 
To be honest I personally do not care about knowing the name of every single farmland, but if I understand [this wiki page](https://wiki.openstreetmap.org/wiki/Tag:place%3Dlocality) correctly, it might still convey useful information in some areas (from what I have seen in France most localities are useless information however).

**Comment by ldsvrn** [comment]
> > Is there a way to simply completely hide theses?
> 
> No, unless you make and compile your own map style (see in `docs/STYLES.md`).
> 
>     * related to [Disable display of (certain) POIs #2355](https://github.com/organicmaps/organicmaps/issues/2355)

Thanks, I'll try making a custom style when I have time.
